hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/stats/ThrottleGauges.java

```diff
@@ -105,18 +104,17 @@ private void registerTypeWith(
     }
 
     private void update(
-            final Instant now,
             final List<DeterministicThrottle> activeThrottles,
             final GasLimitDeterministicThrottle gasThrottle,
             final Map<String, DoubleGauge> namedMetrics) {
         activeThrottles.forEach(throttle -> {
             final var name = throttle.name();
             if (namedMetrics.containsKey(name)) {
-                namedMetrics.get(name).set(throttle.percentUsed(now));
+                namedMetrics.get(name).set(throttle.instantaneousPercentUsed());
```

Comment:
Shouldn't this change just be done for `handleThrottling.allActiveThrottles(), handleThrottling.gasLimitThrottle()` ?

Reply:
After thinking about it, I couldn't see any reason to treat frontend throttles differently. 🤔

Reply:
I see . Makes sense